src/Microsoft.ML.CodeGenerator/Templates/Console/Annotation.tt

```diff
@@ -0,0 +1,19 @@
+﻿<#@ output extension=".txt" #>
+<#+
+void CLI_Annotation()
+{#>
+//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
+//*                                                                                       *
+//*****************************************************************************************
+<#+
+}
+#>
+<#+
+void MB_Annotation()
+{#>
+// This file was auto-generated by ML.NET Model Builder. 
```

Comment:
⚠️ This is missing the `<auto-generated/>` comment, which means diagnostics will be reported in the file even though it is generated.